cnxpublishing/subscribers.py

```diff
@@ -164,7 +167,7 @@ def baking_processor(self, module_ident, ident_hash, cursor=None):
                 logger.exception('Uncaught exception during baking')
                 update_module_state(cursor, module_ident, state, recipe_id)
                 raise
-        finally:
+        else:
```

Comment:
Why did this get changed to an `else` rather than the `finally`? The `finally` ensures the log message is printed and more importantly the state is updated regardless of an error.

Reply:
Aha, I see this actually corrects and issue not seen in this changeset.